.gitignore

```diff
@@ -15,4 +15,7 @@ $RECYCLE.BIN/
 *.msp
 
 # Local config AChecker
-include/config.inc.php
\ No newline at end of file
+include/config.inc.php
+
+# Vendor Folder
```

Comment:
This 2 lines for excluding /vendor folder is un-necessary as PHP Composer is not introduced in this branch.